### PR TITLE
Implement Visible/Invisible Action for BlockListHeaderActions

### PR DESCRIPTION
**Pull request by max-debug022**
**Changes:**
- New Visible & Invisible Actions for BlockListHeaderActions
- Behavior changes of the Toolbar Action Icons

**Details:**
In this commit, the code for handling block actions in the Block List has been refactored. The new `handleToggleVisibilityOfAllSelectedBlocks` function allows toggling the visibility of selected blocks, with "Visible" and "Invisible" icons added for this purpose. 

The "Delete," "Copy," and "Paste" icons remain unchanged in their functionality but have been updated to align with the design specified in the Figma files.  The icons are now always shown regardless of the amount of selected blocks, but they will be disabled when no blocks are selected. 

**Screencaptures:**
**Before:**
Screenshot 2023-08-03 at 08 53 06
Screenshot 2023-08-03 at 08 53 13
**After:**
Screenshot 2023-08-03 at 08 51 55
Screenshot 2023-08-03 at 08 52 04
Screenshot 2023-08-03 at 08 52 14

### CODE REVIEW

**Review by johnnyomair — APPROVED** [review]

**Comment by thomasdax98** [comment]
Please don't merge before @jamesricky approved

**Review by jamesricky — APPROVED** [review]

**Pull request merged**
